src/main/java/de/szut/employees/EmployeeAPI.java

```diff
@@ -0,0 +1,56 @@
+package de.szut.employees;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import de.szut.employees.dto.EmployeeResponseDTO;
+import de.szut.lf8_project.exceptionHandling.InvalidDataException;
+import de.szut.lf8_project.utils.HttpsRequests;
+import lombok.AllArgsConstructor;
+import okhttp3.Request;
+import okhttp3.Response;
+
+import java.io.IOException;
+
+@AllArgsConstructor
+public class EmployeeAPI {
+    private static EmployeeAPI instance;
+    private final HttpsRequests client = new HttpsRequests();
+    private final String baseEmployeesURL = "https://employee.szut.dev/employees";
+    private final ObjectMapper mapper = new ObjectMapper();
```

Comment:
```suggestion
    private final HttpsRequests client = new HttpsRequests();
    private final ObjectMapper mapper = new ObjectMapper();
    private final String baseEmployeesURL = "https://employee.szut.dev/employees";
```

Reply:
Not sure why this change is neccessary, i mean if it is to beatify the code its not even fully correct, the mapper should be at the very top as its the smallest variable.

Reply:
Ich weiß nicht was du meinst :)